Anchor.toml

```diff
@@ -13,7 +13,7 @@ url = "https://api.apr.dev"
 
 [provider]
 cluster = "Localnet"
-wallet = "/home/user/.config/solana/id.json"
+wallet = "/Users/frankfinnegan/.config/solana/id.json"
```

Comment:
didn't know you were a dead Canadian hockey player :)

we should replace this with '~' so that it works for both of us